script/fetch_content.py

```diff
@@ -4,27 +4,47 @@
 import re
 
 
-def arguments():
+def fetch_arguments():
+  """Fetch arguments from command line"""
+
   parser = argparse.ArgumentParser()
-  parser.add_argument('--url', help='add url')
-  parser.add_argument('--text', help='add text')
-  args = parser.parse_args()
-  return args
-def add_characters():
-  for m in re.finditer(text, r.text):
-    start = m.start()
-    end = m.end()
-    new_text = r.text[start-10:end+10]
-    print(new_text)
-args = arguments()
-url = args.url
-text = args.text
-if not url:
-  url = input('Enter the URL:')
-if not text:
-  text = input('Enter the search phrase: ')
-r = requests.get(url)
-if text in r.text:
-  add_characters()
-else:
-  print("Text '" + text + "' not found")
+  parser.add_argument('url', help='add url')
+  parser.add_argument('search_phrase', help='add text')
+  args = parser.parse_args()  
+  return (args.url, args.search_phrase)
+
+
+def print_10_characters_before_and_after_text(search_phrase, text):
+  """
+  Print text with  additional characters.
+
+  This function retrieves the text and returns it with an additional ten characters before and after the text.
+
+  parameters:
+  text: text that someone enters
+  r.text: all text on the website
+  start: index of the first letter of the entered text
+  end: index of the last letter of the entered text
+  new_text: a string with additional characters added before start and after end.
+
+  Return:
+  string: text with 10 additional characters before and after.
+  
+  """
+   
+  if search_phrase in r.text:
```

Comment:
It doesn't help :-)
It should be something like this:
```python
pos = text.find(search_phrase)
if pos >= 0:
  # Phrase found 
else:
  # Phrase not found
```
Check function String.find()